### PR TITLE
feat(FOROME-839): Added Import/Export dataset UI support

### PR DESCRIPTION
**Pull request by AndreyUstyumenko**
https://quantori.atlassian.net/browse/FOROME-839

### CODE REVIEW

File: src/components/simple-popper-modal/simple-popper-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useRef } from 'react'
+import cn, { Argument } from 'classnames'
+
+import { useOutsideClick } from '@core/hooks/use-outside-click'
+
+export interface ISimplePopperModalProps {
+  close: () => void
```
Comment (AlMaQntr) [inline src/components/simple-popper-modal/simple-popper-modal.tsx]: May be `onClose`

File: src/components/export-panel.tsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-export const ExportPanel = ({ close }: Props): ReactElement => {
-  const ref = useRef<any>(null)
-
+import {
+  ISimplePopperModalProps,
+  SimplePopperModal,
+} from './simple-popper-modal/simple-popper-modal'
+import { SimplePopperModalItem } from './simple-popper-modal/simple-popper-modal-item'
+
+export const ExportPanel = ({
```
Comment (AlMaQntr) [inline src/components/export-panel.tsx]: It's modal, may be we can add suffix `Modal` to component name? Like `ExportModal` or `ExportPanelModal`?

File: src/components/popper-modal/PopperModal.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react/display-name */
```
Comment (AlMaQntr) [inline src/components/popper-modal/PopperModal.tsx]: PopperModal.tsx -> popper-modal.tsx

File: src/components/portal/portal.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react'
+import { createPortal } from 'react-dom'
+
+const mount = document.getElementById('portal-root')!
```
Comment (AlMaQntr) [inline src/components/portal/portal.tsx]: I think we can add this element in runtime when Portal used first time

File: src/components/portal/portal.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+const el = document.createElement('div')
+
+export const Portal = ({ children }: React.PropsWithChildren<{}>) => {
+  useEffect(() => {
+    mount.appendChild(el)
```
Comment (AlMaQntr) [inline src/components/portal/portal.tsx]: What will happen if there two <Portals /> on the page at same time?
May be we can create portal element on first mount? 
Something like this:
```
  const portalRootRef = useRef<HTMLDivElement>()
  if (!portalRootRef.current) {
    portalRootRef.current = document.createElement('div')
    portalsRoot.appendChild(portalRootRef.current)
  } 
  
  useEffect(() => () => { portalRootRef.current?.remove() })
  
  return createPortal(children, portalRootRef.current)
```

**Review by AlMaQntr — APPROVED** [review]

File: src/components/simple-popper-modal/simple-popper-modal.tsx (hunk 1 above)
Comment (AndreyUstyumenko) [reply src/components/simple-popper-modal/simple-popper-modal.tsx]: It leads to deeper refactoring and causes too many side-changes for a single PR. 'close' handler is created in a custom hook and it is used in various places among the project.

File: src/components/portal/portal.tsx (hunk 4 above)
Comment (AndreyUstyumenko) [reply src/components/portal/portal.tsx]: What for?

File: src/components/portal/portal.tsx (hunk 5 above)
Comment (AndreyUstyumenko) [reply src/components/portal/portal.tsx]: Doesn't seem to be working as expected

File: src/components/portal/portal.tsx (hunk 4 above)
Comment (AlMaQntr) [reply src/components/portal/portal.tsx]: Remove dependency from index.html

File: src/components/portal/portal.tsx (hunk 5 above)
Comment (AlMaQntr) [reply src/components/portal/portal.tsx]: But you don't prevent this behaviour (

**Review by QYaroslavTrefilov — APPROVED** [review]

**Comment by QYaroslavTrefilov** [comment]
image
I would add `cursor: pointer`  for "Upload file" btn on hover

**Pull request merged**